src/components/URLInput/URLInput.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import Grid from "@material-ui/core/Grid";
-import Textfield from "@material-ui/core/Textfield";
+import Textfield from "@material-ui/core/TextField";
```

Comment:
@GCJyo Found the problem with the pipeline: component locations are case-sensitive, it had to be `TextField` not `Textfield`.

Reply:
Noted, does the merge update the Master